pkgs/development/python-modules/glean-parser/default.nix

```diff
@@ -8,43 +8,39 @@
   jinja2,
   jsonschema,
   pytestCheckHook,
-  pythonOlder,
   pyyaml,
   setuptools,
   setuptools-scm,
-  yamllint,
 }:
 
 buildPythonPackage rec {
   pname = "glean-parser";
-  version = "14.1.2";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.8";
+  version = "14.3.0";
+  pyproject = true;
 
   src = fetchPypi {
     pname = "glean_parser";
     inherit version;
-    hash = "sha256-OL59Tg+rD4M0DjQnkU4IqGMcf6sIioxg6bVDyrbqgww=";
+    hash = "sha256-tI1kMCn7gksLdq2ytKAOiKSd5OxHmsnFrdUsUR6b5IE=";
   };
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace "pytest-runner" "" \
-      --replace "MarkupSafe>=1.1.1,<=2.0.1" "MarkupSafe>=1.1.1"
+      --replace-fail "pytest-runner" ""
   '';
```

Comment:
Can you use `pythonRemoveDeps` for this?

Reply:
I've tried using pythonRemoveDeps, and I always get a missing dependency error for the package.